js/styles/_button.scss

```diff
@@ -219,29 +204,39 @@ $button-shadow-size: $govuk-border-width-form-element;
     color: $govuk-warning-button-text-colour;
   }
 
-  // alphagov/govuk_template includes a specific a:link:focus selector
-  // designed to make unvisited links a slightly darker blue when focussed, so
-  // we need to override the text colour for that combination of selectors so
-  // so that unvisited links styled as buttons do not end up with dark blue
-  // text when focussed.
-  @include govuk-compatibility(govuk_template) {
-    &:link:focus {
-      color: $govuk-warning-button-text-colour;
+  &:hover {
+    background-color: $govuk-warning-button-hover-colour;
+
+    &[disabled] {
+      background-color: $govuk-warning-button-colour;
     }
   }
+}
+
+.cbanner-govuk-button--inverse {
+  background-color: $govuk-inverse-button-colour;
+  box-shadow: 0 $button-shadow-size 0 $govuk-inverse-button-shadow-colour;
+
+  &,
+  &:link,
+  &:visited,
+  &:active,
+  &:hover {
+    color: $govuk-inverse-button-text-colour;
+  }
 
   &:hover {
-    background-color: $govuk-warning-button-hover-colour;
+    background-color: $govuk-inverse-button-hover-colour;
 
     &[disabled] {
-      background-color: $govuk-warning-button-colour;
+      background-color: $govuk-inverse-button-colour;
     }
   }
 }
 
 .cbanner-govuk-button--start {
   @include govuk-typography-weight-bold;
-  @include govuk-typography-responsive($size: 24, $override-line-height: 1);
+  @include govuk-font-size($size: 24, $line-height: 1);
```

Comment:
@ellamdav it seems like govuk-font-size doesn't set the font-family https://github.com/alphagov/govuk-frontend/blob/1871898df79e4c849f89d58f7bb10fb400aed436/packages/govuk-frontend/src/govuk/helpers/_typography.scss#L160-L237 guess they made it more specialised - I think we need just `govuk-font` https://github.com/alphagov/govuk-frontend/blob/1871898df79e4c849f89d58f7bb10fb400aed436/packages/govuk-frontend/src/govuk/helpers/_typography.scss#L253

Reply:
oh nevermind, spotted govuk-font is used further up 🤦‍♂️